docker-compose.yml

```diff
@@ -1,10 +1,10 @@
-version: "3.9"
 services:
   supporters:
-    build: .
+    build:
+      context: .
+      target: server
```

Comment:
Adding build stages let me debug the image before it tried to run rails commands.